demisto_sdk/commands/validate/tests/test_tools.py

```diff
@@ -155,20 +155,23 @@ def create_ps_integration_object(
 
 
 def create_script_object(
-    paths: Optional[List[str]] = None, values: Optional[List[Any]] = None
+    paths: Optional[List[str]] = None,
+    values: Optional[List[Any]] = None,
+    pack_name: Optional[str] = None,
 ):
     """Creating an script object with altered fields from a default script yml structure.
 
     Args:
         paths (Optional[List[str]]): The keys to update.
         values (Optional[List[Any]]): The values to update.
+        pack_name (str): The name of the pack that the script will be inside of
```

Comment:
I've also added something similar to [integration](https://github.com/demisto/demisto-sdk/pull/3964), maybe let's consider syncing.